designcompose/src/main/java/com/android/designcompose/MaterialThemeValues.kt

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.android.designcompose
+
+import androidx.compose.ui.graphics.Color
+
+// Collection name for material theme generated from Material Theme Builder plugin
+const val MATERIAL_THEME_COLLECTION_NAME = "material-theme"
+
+// Material theme variable names for variables generated from Material Theme Builder plugin
+object MaterialThemeConstants {
```

Comment:
There are no strings in there. I need strings to match with the variable names created in Figma.

Reply:
I feel like we could use something like https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Class.html#getDeclaredFields() to parse them out automatically, though that would be a little hacky.

Reply:
Yeah, better not to use reflection if we can avoid it -- it's more surprising than a static table of strings since it can change from a dependency update, but doesn't necessarily show up as a reference.